### PR TITLE
Use Blade for test history and a Console.log tab

### PR DESCRIPTION
**Pull request by stoyan**
Currently commented out behind `// uncomment for Blade` because I need a better setup to test more thoroughly the logged in case.

But yes, it's working! Thanks to @jefflembeck's templates and because Blade lets you drop to just PHP I was able to reuse headers/footers/common as-is. And so no need for more competing code (see issue #2193)

### CODE REVIEW

**Comment by stoyan** [comment]
The second diff resurrects the console.log part of this older page: https://www.webpagetest.org/screen_shot.php?test=220728_AiDc4D_D1J

It's a new tab in the results with a URL like `...org/result/220728_AiDc4D_D1J/1/consolelog/`:
Screen Shot 2022-08-03 at 7 30 35 PM

This page is a bit of a Frankenstein of all templates:
1. The new Blade stuff for the actual content
2. The oldest `www/header.inc` for the results header (dropdown, screenshot, test details)
3. Everything else from the newer `www/templates` (to be deprecated in favor of Blade)

File: www/resources/views/pages/consolelog.blade.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    div.scrollable {
+        max-width: 100em;
+        margin: 0;
+        padding: 0;
+        overflow: auto;
+    }
+
+    tr.even {
+        background: whitesmoke;
```
Comment (deathbearbrown) [inline www/resources/views/pages/consolelog.blade.php:39]: how do you know all these color names??

File: www/resources/views/pages/testhistory.blade.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                <p>Test history is available for up to 30 days as long as your storage isn't cleared. By registering for a free account, you can keep test history for longer, compare tests, and review changes. Additionally, you will also be able to post on the <a href="https://forums.webpagetest.org">WebPageTest Forum</a> and contribute to the discussions there about features, test results and more.</p>
+                <a href="{{ $protocol }}://{{ $host }}/signup" ?>" class="btn-primary">Get Free Access</a>
+            </div>
+        @endif
+        <div class="history_filter">
+            <label for="filter">Filter test history:</label>
+            <input id="filter" name="filter" type="text" onkeyup="filterHistory()" placeholder="Search">
+            @if ($is_logged_in)
+                <label for="days" class="a11y-hidden">Select how far back you want to see</label>
+                <select name="days">
```
Comment (deathbearbrown) [inline www/resources/views/pages/testhistory.blade.php:21]: needs `id="days"` to link to `for`

File: www/resources/views/pages/testhistory.blade.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                    <option value="365" @if ($days === 1) selected @endif>356 Days</option>
+                </select>
+            @endif
+        </div>
+    </form>
+</div>
+<div class="box">
+    <form name="compare" method="get" action="/video/compare.php">
+        <div class="history-controls">
+            <input id="CompareBtn" type="submit" value="Compare Selected Tests">
```
Comment (deathbearbrown) [inline www/resources/views/pages/testhistory.blade.php:35]: add a11y-hidden label for this input

File: www/resources/views/pages/consolelog.blade.php [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                    <div class="results_and_command">
+                        <div class="results_header">
+                            <h2>Console Log</h2>
+                        </div>
+                    </div>
+                    <div id="result" class="results_body">
+
+                        <div class="overflow-container">
+                            <table id="console-log" class="translucent">
+                                <tr>
```
Comment (deathbearbrown) [inline www/resources/views/pages/consolelog.blade.php:60]: thead, tbody

File: www/resources/views/pages/testhistory.blade.php [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+                        <th class="date">Run Date</th>
+                        <th class="location">Run From</th>
+                        <th class="label">Label</th>
+                    </tr>
+                </thead>
+                @if ($is_logged_in)
+                    <tbody id="historyBody">
+                        @foreach ($test_history as $record)
+                            <tr>
+                                <th><input type="checkbox" name="t[]" value="{{ $record->getTestId() }}" /></th>
```
Comment (deathbearbrown) [inline www/resources/views/pages/testhistory.blade.php:52]: a11y hidden label needed

File: www/resources/views/pages/consolelog.blade.php [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+        background: whitesmoke;
+    }
+</style>
+@endsection
+
+@section('content')
+<?php $even = false; ?>
+<div id="test_results-container">
+    <div id="test-1" class="test_results">
+        <div class="test_results-content">
```
Comment (deathbearbrown) [inline www/resources/views/pages/consolelog.blade.php:48]: this is legit just a comment, but dang that's a lot of wrapper divs

**Review by deathbearbrown — APPROVED** [review]

File: www/resources/views/pages/consolelog.blade.php (hunk 1 above)
Comment (stoyan) [reply www/resources/views/pages/consolelog.blade.php:39]: haha, I don't, unfortunately. They do seem handy :) 
The css+html is a copy of the old code, that's currently not reachable: https://github.com/WPO-Foundation/webpagetest/blob/master/www/screen_shot.php#L88
So the purpose of the diff is two-fold: to bring back the feature and while Blade-ing

**Pull request merged**